_docs/_latest/setup/forseti-on-gke.md

```diff
@@ -0,0 +1,306 @@
+
+1. Forseti has been deployed in a GCP project.
+2. A GKE cluster has already been deployed in GCP under the same organization as Forseti.
+3. The [GKE credentials](https://cloud.google.com/kubernetes-engine/docs/how-to/cluster-access-for-kubectl) exist on the machine executing the deployement
+4. The [Helm binary](https://helm.sh/docs/using_helm/) is installed and initiallized on the machine executing the deployment.
+
+After these preconditions are met, add the *forseti-security-charts* Helm repo to your Helm environment.
+
+```bash
+helm repo add forseti-security https://forseti-security-charts.storage.googleapis.com/release
+```
+
+Create and download [IAM service account keys](https://cloud.google.com/iam/docs/creating-managing-service-account-keys) in JSON format for the **forseti-server** and **forseti-client** service accounts.
+
+Follow the [chart installation instructions](https://github.com/forseti-security/helm-charts/tree/master/charts/forseti-security#installing-the-forseti-security-chart) to install Forseti on GKE.
+
+
+## Post Deployment Configuration Changes
+
+After deploying Forseti, the default server configuration defines
+
+```yaml
+    rules_path: /home/ubuntu/forseti-security/rules
+```
+
+This path doesn't exist in the container.  Instead, the value of `rules_path` must point to the rules path in the Forseti server GCS bucket.  The can be changed by executing the following.
+
+1. Obtain the *forseti_server_conf.yaml* file from the GCS bucket.
+
+```bash
+gsutil cp gs://[FORSETI_SERVER_BUCKET]/configs/forseti_server_conf.yaml forseti_server_conf.yaml
+```
+
+2. Edit the forseti_server_conf.yaml file so that the `rules_path` points to the rules folder in the GCS bucket.
+
+```yaml
+    rules_path: gs://[FORSETI_SERVER_BUCKET]/rules
+```
+
+3. Save the file and exit the editor.
+
+4. Copy the *forseti_server_conf.yaml* file back to GCS.
+
+```bash
+gsutil cp forseti_server_conf.yaml gs://[FORSETI_SERVER_BUCKET]/configs/forseti_server_conf.yaml
+```
+
+5. Ensure the *Content-Type* of the *forseti_server_conf.yaml* is `text/plain`.
+```bash
+gsutil setmeta -h "Content-Type:text/plain; charset=utf-8" gs://[FORSETI_SERVER_BUCKET]/configs/forseti_conf_server.yaml
+```
+
+### Reload Forseti on GKE
+
+**Note:** If any changes are made to the *forseti_server_conf.yaml* file in GCS, one of the following steps is necessary.  In a future version of this feature, this will be automated.
+
+#### With Terraform
+
+```bash
+terraform apply
+```
+
+#### With Helm
+```
+helm upgrade -i forseti forseti-security/forseti-security \
+    --set production=true \
+    --recreate-pods \
+    --set-string serverKeyContents="$(cat forseti-server.json | base64 - -w 0)" \
+    --set-string orchestratorKeyContents="$(cat forseti-client.json | base64 - -w 0)" \
+    --set-string serverConfigContents="$(gsutil cat gs://<BUCKET_NAME>/<CONFIGS_FOLDER>/forseti_conf_server.yaml | base64 -)" \
```

Comment:
Do we need to indicate `<CONFIGS_FOLDER>` or is `configs` enough?

Reply:
I could go either way.

Reply:
Let's change it to `configs` since I don't think there should be any differences in the path among users. Less work!